raft/progress.go

```diff
@@ -56,6 +56,11 @@ type Progress struct {
 	// is reported to be failed.
 	PendingSnapshot uint64
 
+	// recentActive is true if the progress is recently active. Receiving any messages
+	// from the corresponding follower indicates the progress is active.
+	// recentActive can be reset to false after an election timeout.
+	recentActive bool
```

Comment:
is name `active` good enough? active always indicates the recent status of the progress.